src/components/Count/Count.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './Count.scss';
+
+const Count = props => {
+  const { number, setNumber } = props;
```

Comment:
```suggestion

const Count = ({number, setNumber}) => {
```
이렇게 축약해서 사용할 수 있습니다~